ledger/catchpointtracker.go

```diff
@@ -929,7 +928,7 @@ func (ct *catchpointTracker) pruneFirstStageRecordsData(ctx context.Context, max
 	for _, round := range rounds {
 		relCatchpointDataFilePath :=
 			filepath.Join(trackerdb.CatchpointDirName, makeCatchpointDataFilePath(round))
-		err = trackerdb.RemoveSingleCatchpointFileFromDisk(ct.dbDirectory, relCatchpointDataFilePath)
+		err = trackerdb.RemoveSingleCatchpointFileFromDisk(ct.tmpDir, relCatchpointDataFilePath)
```

Comment:
Okay, so this is a cleanup function for intermediate files and the tmpDir is the build location.